config/config.go

```diff
@@ -34,6 +39,10 @@ var (
 // Validate creates the list of implicit dependencies.
 func (config *Config) Validate(path string) ([]string, error) {
 	cameraName := ""
+	if config.CloudStoryEnabled {
+		return config.ValidateCloudStoryEnabled(path)
```

Comment:
does this need to return here? can we just check our params exist then run the remainder of validate?

Reply:
updated to reflect changes we talked about offline